benchmarks/dense_matmul/genbenchmark.py

```diff
@@ -128,10 +135,67 @@ def output_log_benchmark(benchmark_name: str, utilization: dict[str, int]) -> st
 
     sizes = list(itertools.product(selected_dims, repeat=3))
 
+    # some other relevant neural network sizes:
```

Comment:
@xiaoling-yi could you maybe add a comment (with a link?) here that says where these numbers come from exactly? I think this would be nice for later reference.
>they are from ["lstm", "mnv2", "resnet18", "vit", 'ibertbase'].

Essentially I don't know what this means, and I'd like a comment in code that explains this rather than a comment in a pull request :smile: